pkg/handlers/get.go

```diff
@@ -0,0 +1,112 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/Ubivius/microservice-text-chat/pkg/data"
+)
+
+func (textChatHandler *TextChatHandler) GetMessages(responseWriter http.ResponseWriter, request *http.Request) {
+	textChatHandler.logger.Println("Handle GET messages")
+	messageList := data.GetMessages()
+	err := json.NewEncoder(responseWriter).Encode(messageList)
+	if err != nil {
+		textChatHandler.logger.Println("[ERROR] serializing message", err)
+		http.Error(responseWriter, "Unable to marshal json", http.StatusInternalServerError)
+	}
+}
+
+func (textChatHandler *TextChatHandler) GetConversations(responseWriter http.ResponseWriter, request *http.Request) {
```

Comment:
Est-ce qu'il y a un moment qu'on va avoir besoins d'avoir toutes les conversations et messages à part pour tester? pcq sinon on pourrait ne pas les mettres (un commentaire que Malcolm m'avait fait dans achievements)

Reply:
We probably shouldn't have an endpoint that allows to fetch everything in the database. We can access the db or the logs if we need to find anything specific